File: inflearn/java/1-03/Main.java
import java.util.*;

public class Main {
    public static String solution(String str) {
        String answer = "";
        int max = 0;
        String[] str_split = str.split(" ");
        for (String s : str_split) {
            int len = s.length();
            if (len > max) {
                max = len;
                answer = s;
            }
        }
        return answer;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String input = sc.nextLine(); // 공백 포함한 문자열 입력

        System.out.println(solution(input));
    }
}

File: inflearn/javascript/1-03/main.js
const ps = require('../modules/ps');

const solution = (input) => {
  let answer = '';
  let max_len = 0;
  input = input.split(' ');
  input.forEach((element) => {
    if (element.length > max_len) {
      answer = element;
      max_len = element.length;
    }
  });
  return answer;
};

ps.main(async () => {
  let input = await ps.readLine(); // 문자열 입력

  console.log(solution(input));
});
